content/en/docs/setup/install/multicluster/primary-remote/index.md

```diff
@@ -16,6 +16,13 @@ Before proceeding, be sure to complete the steps under
 [before you begin](/docs/setup/install/multicluster/before-you-begin).
 
 {{< boilerplate multi-cluster-with-metallb >}}
+{{< warning >}}
```

Comment:
Does it not work on AWS if you follow the tip [in this section](https://istio.io/latest/docs/setup/install/multicluster/primary-remote/#configure-cluster2-as-a-remote)?

![image](https://github.com/istio/istio.io/assets/2752495/58696295-2562-4e75-a204-e02d355fed20)

Reply:
thanks @frankbu 

few points here:
- the issue was brought to my attention by AWS where their enterprise customer was failing - from the current instructions AWS and myself had to reproduce the customer environment - which was significant amount of time and adding these wording should have some time to people who's planning the deployment in AWS
- personally I was not able to get External Control Plane working on AWS - opened [this issue](https://github.com/istio/istio/issues/46573) - it seems a different issue but I might be missing something.